src/contracts/constants.ts

```diff
@@ -1,4 +1,4 @@
-import { getConfig } from 'config'
+import { getConfig } from '../config'
```

Comment:
revert me

Reply:
@kaladinlight are you able to build the assets locally without this? I wasn't

Reply:
You are correct, my bad. As per discussion in dms, will follow up with https://github.com/shapeshift/web/issues/7300, and revert back to the previous import when tackling that issue

Reply:
Nice one @NeOMakinG! Looks like we've been borked for 2 weeks on regen script too, so this should make things happy again! 🎉 
https://github.com/shapeshift/web/actions/workflows/generate.yml